mesh/profiles/views.py

```diff
@@ -56,17 +56,7 @@ def post(self, request, account_id, *args, **kwargs):
 
       Returns a JSON response containing the saved biography of the specified profile through id.
       """
-      data = json.loads(request.body)
-
-      if 'accountID' not in data:
-          return JsonResponse({'error': 'Invalid request. Missing accountID field.'}, status=401)
-      if 'biography' not in data:
-          return JsonResponse({'error': 'Invalid request. Missing biography field.'}, status=400)
-      else:
-          user = Profile.objects.get(accountID=account_id)
-          user.biography = data["biography"]
-          user.save()
-          return JsonResponse({'message': "Biography saved successfully."}, status=200)
+      return post_data(account_id, "biography", request)
```

Comment:
I'm aware that the post request for biography was outdated, does that means we need to make a new task for handling to post request for biography?

Reply:
No, I fixed the post request for biography in this PR.